samples/excel/90-just-for-fun/patterns.yaml

```diff
@@ -0,0 +1,210 @@
+id: excel-90-just-for-fun-patterns-
+name: Colorful Patterns
+description: Shows how to use range formatting to draw interesting pattern. Contributed by Alexander Zlatkovski
+author: AlexanderZlatkovski
+host: EXCEL
+api_set:
+    ExcelApi: 1.2
+script:
+    content: |-
+        $("#spiral").click(drawSpiral);
+        $("#squares").click(drawSquares);
+        $("#cross").click(drawCross);
+
+        async function drawSquares() {
+            try {
+                await Excel.run(async (context) => {
+                    const sheet = await OfficeHelpers.ExcelUtilities.forceCreateSheet(context.workbook, "Patterns", true /*clearOnly*/);
+                    sheet.activate();
```

Comment:
@jakobpn , created pull request https://github.com/OfficeDev/office-js-helpers/pull/30.
Once approve and checked in, the samples can go in as is -- except we'll need to bump the version number of OfficeJsHelpers from 0.7.1 to 0.7.2.  However, I think we want to do it all-up for all samples.  So my suggestion is that once OfficeJsHelpers is patched, you approve and merge in this pull request, and then I can follow up with a different pull request that updates all the samples to 0.7.2 in bulk.